libvips/foreign/webpsave.c

```diff
@@ -1161,6 +1190,8 @@ vips_foreign_save_webp_mime_init(VipsForeignSaveWebpMime *mime)
  * * @near_lossless: %gboolean, preprocess in lossless mode (controlled by Q)
  * * @alpha_q: %gint, set alpha quality in lossless mode
  * * @effort: %gint, level of CPU effort to reduce file size
+ * * @target_size: %gint, desired target size in bytes
+ * * @pass: %gint, number of entropy-analysis passes
```

Comment:
And add a paragraph to the doc text below explaining how `target_size` works.

Reply:
Let me know if I need to update the verbiage/phrasing.